.vscode/settings.json

```diff
@@ -12,5 +12,6 @@
     "resolvePluginsRelativeTo": "./node_modules/@ridedott/eslint-config"
   },
   "eslint.run": "onType",
-  "jest.autoEnable": false
+  "jest.autoEnable": false,
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
This ensures to use the typescript version of the installed node module and not that of vs code itself.